src/onnx/parse_convolution.cpp

```diff
@@ -149,9 +149,40 @@ struct parse_convolution : op_parser<parse_convolution>
 
         recalc_conv_attributes(values, kdims);
 
+        instruction_ref ret;
+        // parse a_zero_point and b_zero_point values
+        auto l0_zp = l0;
+        auto w_zp =  weights;
+
         op.from_value(values);
-        auto l1 = info.add_instruction(op, l0, args[1]);
-        return info.add_bias(args, l1, 1);
+        if (op.name() == "quant_convolution") {
+            if(args.size() > 2)
+            {
+                l0_zp = args[2];
+                if (l0_zp->get_shape().type() != l0_shape.type()) {
+                    MIGRAPHX_THROW("PARSE: ConvInteger Data and Data Zero Point must have same type");
+                }
+
+                l0_zp = info.add_common_op("sub", l0, l0_zp);
```

Comment:
Could you please also check that it is not zero.

Reply:
Just to be clear, your concern is if the zero point exists but it itself is zero thus avoid adding subtraction node + broadcasts?

Reply:
Yes, please. It is as if no Zero Point was specified, and the default case would kick in. Thanks.